src/pal/src/exception/seh-unwind.cpp

```diff
@@ -491,7 +491,17 @@ BOOL PAL_VirtualUnwindOutOfProc(CONTEXT *context,
 
     LibunwindCallbacksInfo.Context = context;
     LibunwindCallbacksInfo.readMemCallback = readMemCallback;
+#if UNWIND_CONTEXT_IS_UCONTEXT_T
     WinContextToUnwindContext(context, &unwContext);
+#else
+    st = unw_getcontext(&unwContext);
```

Comment:
It seems we only pass in the unwContext to GetContextPointers to filter out its own location on the stack.  That shouldn't matter for the remote case.

Do we even need to do this for the remote case?  Maybe we should pass in NULL for unwContext for this case, and handle that properly in GetContextPointers?